README.md

```diff
@@ -1,89 +1,55 @@
-<p align="center">
-  <img src="https://raw.githubusercontent.com/slatedocs/img/master/logo-slate.png" alt="Slate: API Documentation Generator" width="226">
-  <br>
-  <a href="https://github.com/slatedocs/slate/actions?query=workflow%3ABuild+branch%3Amaster"><img src="https://github.com/slatedocs/slate/workflows/Build/badge.svg?branch=master" alt="Build Status"></a>
-</p>
+# Skynet SDK Docs
 
-<p align="center">Slate helps you create beautiful, intelligent, responsive API documentation.</p>
+Documentation for the Skynet SDKs in various languages.
 
-<p align="center"><img src="https://raw.githubusercontent.com/slatedocs/img/master/screenshot-slate.png" width=700 alt="Screenshot of Example Documentation created with Slate"></p>
+## Status of SDKs
```

Comment:
pretty good format over all, at least it's a starting point to consolidate a list of issues we can then prioritize 
perhaps that should be an action point... if we would list a whole bunch of issues that we can then prioritize as a team that 'd be good I think